docs/source/tabs/labeled_diagnostics.rst

```diff
@@ -1,7 +1,7 @@
-.. _tab_labeled_diagnostics:
+.. _tab_image_diagnostics:
```

Comment:
go ahead and leave this as `_tab_labeled_diagnostics` as before, otherwise we need to find and update all references to this tag in the docs and update them as well